build.sh

```diff
@@ -93,8 +93,11 @@ cd ../..
 
 echo ""
 echo "Copying extra plugins..."
-cp -r plugins/* ckeditor/plugins/
-
+if [ "$(uname)" == "Darwin" ]; then
+	cp -R plugins/ ckeditor/plugins/
```

Comment:
Man for Linux cp (GNU coreutils) 8.2 shows:
```
-R, -r, --recursive
              copy directories recursively
```
So it should work with `-R` for both OSX and Linux.